src/Struct/FindologicProduct.php

```diff
@@ -1012,4 +1020,26 @@ protected function getAttributeKey(?string $key): ?string
 
         return $key;
     }
+
+    protected function decodeHtmlEntities(array $values): array
+    {
+        foreach ($values as $key => $value) {
+            $values[$key] = $this->decodeHtmlEntity($value);
+        }
+
+        return $values;
+    }
+
+    /**
+     * @param mixed $value
+     * @return string|mixed
+     */
+    protected function decodeHtmlEntity($value)
+    {
+        if (!is_string($value)) {
+            return $value;
+        }
+
+        return html_entity_decode($value);
```

Comment:
The fix is fine, but shouldn't `libflexport` be handling such cases already?

Reply:
Why would it? It can be a legitimate case, for an own, or different shopsystem and it limits the developer. Maybe as an option, but even that feels out-of-scope for a library that wraps data into a Findologic-consumable XML/CSV.

Reply:
Now we need to make sure to call method `decodeHtmlEntity` for every Attribute creation. If a new attribute gets added we might miss it, if libflexport handles it, there is no way to forget it, right?

Reply:
Yes but it is in our responsibility to create a wrapper around libflexport to ensure this is always being done. I really don't see this job at libflexport.